package.json

```diff
@@ -4,39 +4,35 @@
     "url": "https://github.com/linterjs/core/issues"
   },
   "dependencies": {
-    "common-tags": "^1.8.0",
-    "loglevel-colored-level-prefix": "^1.0.0",
-    "read-pkg-up": "^4.0.0",
-    "require-relative": "^0.8.7",
-    "tslib": "^1.9.3"
+    "common-tags": "~1.8.0",
+    "loglevel-colored-level-prefix": "~1.0.0",
+    "read-pkg-up": "~7.0.1",
+    "require-relative": "~0.8.7",
+    "tslib": "~2.0.0"
   },
   "description": "Lint all the things... with one command",
   "devDependencies": {
-    "@babel/core": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
-    "@types/jest": "^23.3.2",
-    "@types/node": "^10.9.4",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-jest": "^23.4.2",
-    "codeclimate-test-reporter": "^0.5.0",
-    "codecov": "^3.1.0",
-    "jest": "^23.5.0",
-    "jest-cli": "^23.5.0",
-    "jest-plugin-fs": "^2.9.0",
-    "jest-plugins": "^2.9.0",
-    "npm-check": "^5.8.0",
-    "prettier": "^1.14.2",
-    "rimraf": "^2.6.2",
-    "ts-jest": "^23.1.4",
-    "tslint": "^5.11.0",
-    "tslint-config-airbnb": "^5.11.0",
-    "tslint-config-prettier": "^1.15.0",
-    "tslint-language-service": "^0.9.9",
-    "tslint-plugin-prettier": "^1.3.0",
-    "typescript": "^3.0.3"
+    "@babel/core": "~7.10.5",
+    "@babel/preset-env": "~7.10.4",
+    "@types/common-tags": "~1.8.0",
+    "@types/jest": "~26.0.7",
+    "@types/node": "~14.0.25",
+    "@types/require-relative": "~0.8.0",
+    "babel-jest": "~26.1.0",
+    "codecov": "~3.7.2",
+    "jest": "~26.1.0",
+    "jest-cli": "~26.1.0",
+    "jest-plugin-fs": "~2.9.0",
+    "jest-plugins": "~2.9.0",
+    "prettier": "2.0.5",
+    "rimraf": "~3.0.2",
+    "ts-jest": "~26.1.3",
+    "typescript": "~3.9.7"
   },
   "engines": {
-    "node": ">=8.0.0"
```

Comment:
Any specific reason for 12.18? Does it have any features we depend on? Can't we just require `>=12.0.0`?

Reply:
Nope, no reasons, just the latest from LTS
Will change it to 12.0 👍